step-templates/aws-deploy-lambda-iam.json

```
@@ -0,0 +1,224 @@
+{
+  "Id": "bdb4c3e3-b3b8-403d-94dc-49898c732197",
+  "Name": "AWS - Deploy Lambda Function via IAM Role",
```

Nitpick suggestion for name
```suggestion
    "Name": "AWS - Deploy Lambda Function using IAM Role auth",
```